.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -0,0 +1,104 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Test Runner Nightly Integration Tests
+
+on:
+  push:
```

Comment:
Do we allow people to push directly to dev?

Reply:
Good question! I think WSM team owns this question. I will add it to the doc for future reference as well.